code/modules/examine/descriptions/weapons.dm

```diff
@@ -0,0 +1,81 @@
+/*
+	Note: This file is meant for actual weapons (guns, swords, etc), and not the stupid 'every obj is a weapon, except when it's not' thing.
+*/
+
+//******
+//*Guns*
+//******
+
+//This contains a lot of copypasta but I'm told it's better then a lot of New()s appending the var.
+/obj/item/weapon/gun
+	description_info = "This is a gun.  To fire the weapon, have your gun mode set to 'fire', \
+	then click where you want to fire."
+
+/obj/item/weapon/gun/energy
+	description_info = "This is an energy weapon.  To fire the weapon, have your gun mode set to 'fire', \
+	then click where you want to fire.  Most energy weapons can fire through windows harmlessly.  To recharge this weapon, use a weapon recharger."
+
+/obj/item/weapon/gun/energy/crossbow
```

Comment:
this needs to be tagged to the kinetic accelerator variety, not this one; this is currently an admin only weapon.